isamples_metadata/taxonomy/OpenContextClassifierInput.py

```diff
@@ -11,14 +11,15 @@ def __init__(self, thing):
         self.OC_material_field = "Consists of_label"
         self.OC_sample_field = "Has type_label"
 
-    def build_text(self, description_map, labelType):
+    def build_text(self, description_map, labelType, description_field):
         """Return the concatenated text of informative fields in the
         description_map"""
         concatenated_text = ""
-        for key, value in description_map.items():
+        for field in description_field:
```

Comment:
this is going to be further updated in a separate branch where I will add updates for OpenContext.